tests/snippets/dict.py

```diff
@@ -37,11 +37,18 @@ def dict_eq(d1, d2):
         res.add(key)
 assert res == set(['a','b'])
 
+x = {'a': 1, 'b': 2, 'c': 3, 'd': 3}
+del x['c']
+it = iter(x.items())
```

Comment:
could we get more tests of these? Including the "view"-ness? Something like

```
data = {1: 2, 3: 4}
items = data.items()
assert list(items) == [(1, 2), (3, 4)]
data[5] = 6
assert list(items) == [(1, 2), (3, 4), (5, 6)]
```

Reply:
I think this is a good idea to add.